counsel.el

```diff
@@ -4191,7 +4191,7 @@ When ARG is non-nil, display all active evil registers."
       (let* ((counsel--mark-ring-calling-point (point))
              (candidates (counsel-mark--get-evil-candidates arg)))
         (if candidates
-            (counsel-mark--ivy-read candidates 'counsel-evil-marks)
+            (counsel-mark--ivy-read "Evil marks: " candidates 'counsel-evil-marks)
```

Comment:
Maybe this prompt should be singular rather than plural, for consistency? Or was this intentional?

Reply:
No, that's not intentional. Thnx for poiting it out. Should be fixed now.